util/chplenv/__init__.py

```diff
@@ -4,6 +4,7 @@
     'chpl_aux_filesys',
     'chpl_make',
     'chpl_platform',
+    'chpl_python_version',
```

Comment:
This should be put down with the `General purpose helpers` instead of the core chplenv scripts